src/sx/src/cssReset.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import csstree from 'css-tree';
+
+function css(strings): string {
+  const css = strings.join('');
+  return csstree.generate(csstree.parse(css));
+}
+
+// TODO: consider including https://github.com/necolas/normalize.css
```

Comment:
I am not sure we should mix reset and normalize, since they are 2 different ideas, but we could add an option for choosing between normalize and reset.

Reply:
I'd add normalisation, I'm already using it

Reply:
Yeah, that is why we could allow the dev to choose between CSS reset and normalize.

Reply:
@tbergq I would not like to make it more complex with additional configuration options and choices. It would be nice if SX provided a good to go default and that's it (so you don't have to add these resets/normalizers on your own). At least for now before finishing the basics.

Reply:
>  I would not like to make it more complex 

Yeah, I am fine with that 👍 

> It would be nice if SX provided a good to go default

Exactly, but I don't think we should include reset and normalize at once.